Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -110,5 +110,47 @@ public GameObject GetControllerModelOverride(Type controllerType, Handedness han
 
             return null;
         }
+
+        /// <summary>
+        /// Gets the override <see cref="IMixedRealityControllerVisualizer"/> type for a specific controller type and hand.
+        /// If the requested controller type is not defined, DefaultControllerVisualizationType is returned.
+        /// </summary>
+        /// <param name="controllerType">The type of controller to query for</param>
+        /// <param name="hand">The specific hand assigned to the controller</param>
+        public SystemType GetControllerVisualizationTypeOverride(Type controllerType, Handedness hand)
+        {
+            for (int i = 0; i < controllerVisualizationSettings.Length; i++)
+            {
+                if (controllerVisualizationSettings[i].ControllerType != null &&
+                    controllerVisualizationSettings[i].ControllerType.Type == controllerType &&
+                    (controllerVisualizationSettings[i].Handedness == hand || controllerVisualizationSettings[i].Handedness == Handedness.Both))
```

Comment:
These conditions are shared between GetControllerVisualizationTypeOverride and GetControllerModelOverride and GetUseDefaultModelsOverride, and probably could be a little helper.

Reply:
changed to SettingContainsParameters